luda-editor/luda-editor-client/src/app/types/camera_angle.rs

```diff
@@ -73,3 +75,9 @@ impl CameraAngle {
         )
     }
 }
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+pub struct Circumscribed {
```

Comment:
It's not a good position to put Circumscribed. Please leave only camera angle related thing in camera_angle.rs. Circumscribed should be in common like app/types/mod.rs or app/types/circumscribed.rs